api-test/ApiConfig.postman_collection.json

```diff
@@ -48,6 +48,46 @@
 					},
 					"response": []
 				},
+				{
+					"name": "createCreditorInstitution",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"createCreditorInstitution\", function () {",
+									"  pm.response.to.have.status(201);",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n  \"creditor_institution_code\": \"0010101010\",\n  \"enabled\": true,\n  \"business_name\": \"Comune di Lorem Ipsum\",\n  \"address\": {\n    \"location\": \"Via delle vie 3\",\n    \"city\": \"Rome\",\n    \"zip_code\": \"00187\",\n    \"country_code\": \"RM\",\n    \"tax_domicile\": \"00111\"\n  },\n  \"fk_int_quadrature\": null,\n  \"psp_payment\": true,\n  \"reporting_ftp\": false,\n  \"reporting_zip\": true\n}",
```

Comment:
Could be better to use a generic random creditor code , to avoid that two consecutive executions of the collections give differential results ( ex. `"status": 409` ) 

In order to do that we can sue postman `Pre-request `
```
function makeid(length) {
    var result           = '';
    var characters       = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
    var charactersLength = characters.length;
    for ( var i = 0; i < length; i++ ) {
      result += characters.charAt(Math.floor(Math.random() * 
 charactersLength));
   }
   return result;
}

pm.globals.set("creditor_institution_code", makeid(35));
```
As next step we shuold extend collection with edge case too, example `creditor_institution_code=""`

Reply:
I have some issues with k6. I am still trying to solve